devsetup/scripts/gen-edpm-node-bgp.sh

```diff
@@ -218,6 +218,9 @@ sed -i /PERSISTENT_DHCLIENT/d /etc/sysconfig/network-scripts/ifcfg-eth0
 systemctl restart NetworkManager
 ip r add default via 192.168.122.1
 
+# Remove stale artifacts that are auto-generated during EDPM deployment
+rm -f /etc/nvme/hostid /etc/nvme/hostnqn
```

Comment:
nit: This is ok, but we could also create the right values instead of making os-brick do it:

```
/usr/sbin/dmidecode -s system-uuid > /etc/nvme/hostid
/usr/sbin/nvme gen-hostnqn > /etc/nvme/hostnqn
```

Reply:
@ASBishop you can follow up with the above.

Reply:
There's no real need to change it, because os-brick will recreate those files.
It's just my preference. I like an explicit recreation instead of just removing them, but everything will work either way.

Reply:
It's unlikely that os-brick will need to recreate the files because the edpm deployment installs the nvme-cli package, and it will recreate them first.